### ISSUE TITLE
Pull to refresh not working reliably in Expo client on projects tab

### ISSUE THREAD

**Issue by brentvatne** [expo/expo#10370]
### Summary of Issue 

Using the latest Expo client for Android: when trying to pull to refresh, I can't get it to show trigger. Pulling down has no effect on the UI.

**Comment by XandruDavid** [comment]
Same problem here. Pull to refresh totally stopped working on Android on expo sdk 39. It isn't only a UI problem, the onRefresh function provided on my FlatList isn't triggered at all.

Everything seems to work fine on iOS.

**Comment by sjchmiela** [comment]
What device/system version were you experiencing the issue on? I was trying to do it on a Nokia 1 Plus, API 29 and I wasn't able to.

**Comment by brentvatne** [comment]
@sjchmiela - this was happening to me on oneplus 6t / api 29 on the play store client, but not on the local build from expo/expo.

**Comment by sjchmiela** [comment]
I've tried reproducing on both a local build and an expo.io APK, but maybe the Play Store client is somewhat different… checking now!

**Comment by XandruDavid** [comment]
I resolved it just now.
As the issue was persistent on all the different android testing devices and simulators I use, your answers made me think it must be a dependency version conflict or something, this made me realise I was importing the FlatList from `react-native-gesture-handler`. Once I changed my imports to use the `react-native` FlatList directly, the pull-to-refresh function started working on android too.
